rust/datafusion/src/physical_plan/group_scalar.rs

```diff
@@ -120,12 +131,9 @@ mod tests {
     use crate::error::DataFusionError;
 
     macro_rules! scalar_eq_test {
-        ($TYPE:expr, $VALUE:expr) => {{
-            let scalar_value = $TYPE($VALUE);
-            let a = GroupByScalar::try_from(&scalar_value).unwrap();
-
-            let scalar_value = $TYPE($VALUE);
-            let b = GroupByScalar::try_from(&scalar_value).unwrap();
+        ($EXPR:expr) => {{
```

Comment:
I don't understand this change -- it seems to now be testing that `try_from` returns the same value for the same input...